tests/python/gaia-ui-tests/gaiatest/apps/contacts/regions/contact_form.py

```diff
@@ -172,6 +172,8 @@ def tap_cancel(self):
 
     def tap_delete(self):
         delete_item = self.marionette.find_element(*self._delete_locator)
+        self.marionette.execute_script(
```

Comment:
Yes, I'm planning on doing that in https://bugzilla.mozilla.org/show_bug.cgi?id=1195341